include/caffe/net.hpp

```diff
@@ -26,6 +27,8 @@ class Net {
   explicit Net(const NetParameter& param);
   explicit Net(const string& param_file, Phase phase,
       const int level = 0, const vector<string>* stages = NULL);
```

Comment:
prefix comma, postfix const

Reply:
> prefix comma, postfix const

This is just an extension to caffe code. We have to use caffe coding style.